recipes-devtools/cargo/cargo/0001-Disable-http2.patch

```diff
@@ -19,11 +19,11 @@ index 8238380861d9..ced1defea459 100644
 --- a/Cargo.toml
 +++ b/Cargo.toml
 @@ -24,7 +24,7 @@ bytesize = "1.0"
- crates-io = { path = "src/crates-io", version = "0.23" }
```

Comment:
Won't changing this patch in place prevent older versions of cargo from building?